Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "stackable-secret-operator"
+license = "OSL-3.0"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
We usually have description, authors and repository as well.
(and maybe remove that comment)

Reply:
8247afb